client/container_list.go

```diff
@@ -2,9 +2,11 @@ package client
 
 import (
 	"encoding/json"
+	"errors"
 	"net/url"
 	"strconv"
 
+	"github.com/docker/docker/pkg/version"
```

Comment:
Okay I understand, since `engine-api` is vendored into Docker? I'll try to find another package, but I suppose long term it would be best to use the same one.